hmda-regulator/src/main/scala/hmda/regulator/query/RegulatorComponent.scala

```diff
@@ -490,14 +490,21 @@ trait RegulatorComponent {
     def deleteByLei(lei: String): Future[Int] = {
       db.run(table.filter(_.lei === lei).delete)
     }
-
     def count(): Future[Int] = {
       db.run(table.size.result)
     }
-
     def getAllLARs(
-        bankIgnoreList: Array[String]): Future[Seq[LarEntityImpl]] = {
-      db.run(table.filterNot(_.lei.toUpperCase inSet bankIgnoreList).result)
+        bankIgnoreList: Array[String]): DatabasePublisher[LarEntityImpl] = {
+      db.stream(
+        table
+          .filterNot(_.lei.toUpperCase inSet bankIgnoreList)
+          .result
+          .withStatementParameters(
+            rsType = ResultSetType.ForwardOnly,
+            rsConcurrency = ResultSetConcurrency.ReadOnly,
+            fetchSize = 1000
```

Comment:
`fetchSize` might work well as part of the configuration incase we decide to tune it down the road.